lib/bootloader_setup.pm

```diff
@@ -81,8 +81,8 @@ sub boot_local_disk {
 sub boot_into_snapshot {
     send_key_until_needlematch('boot-menu-snapshot', 'down', 10, 5);
     send_key 'ret';
-    # wait to avoid send down key early in grub_test_snapshot.
-    wait_still_screen 1;
+    # assert needle to avoid send down key early in grub_test_snapshot.
+    assert_screen 'snap-default' if check_var('ARCH', 'ppc64le');
```

Comment:
please make that `if check_var('OFW')` because it is not architecture dependant but firmware dependant. This way also the "ppc64" tests on openSUSE - not ppc64le - should work as well

Reply:
Updated. Thanks for the comments.